QModManager/API/SMLHelper/Utility/SaveUtils.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace QModManager.API.SMLHelper.Utility
+{
+    using System.IO;
+
+    /* TODO: Maybe add more saving related functions here,
+       such as those related to serializing? */
+    public static class SaveUtils
+    {
+        /// <summary>
+        /// Returns the path to the current save slot's directory.
+        /// </summary>
+        public static string GetCurrentSaveDataDir()
+        {
+            return Path.Combine(SNUtils.savedGamesDir, Utils.GetSavegameDir());
```

Comment:
In https://github.com/PrimeSonic/PrimeSonicSubnauticaMods/commit/f748e904186367f26159f2568be918803b60fc6b, you replaced this with something else. Should this also be replaced here?

Reply:
That change is only relevant to Below Zero.  
Any change applied here will need to make the distinction since Subnautica and BZ have different save file locations.